Registry/Registry.EM_COMMON

```diff
@@ -2611,6 +2629,7 @@ rconfig   integer morr_rimed_ice          namelist,physics      1              1
 
 rconfig   integer clean_atm_diag          namelist,physics      1              0       rh    "clean_atm_diag"     "option to switch on clean sky diagnostics (for chem)"   "flag"
 rconfig   integer calc_clean_atm_diag     derived               1              0       -     "calc_clean_atm_diag" "carries decision on using clean sky diagnostics"       "flag"
+rconfig   integer acc_phy_tend            namelist,physics      1              0       rh    "acc_phy_tend"       "option to output accumulated physics tendencies"   "flag"
```

Comment:
@weiwangncar 
What do you think about making this a max_domains option? Maybe there is interest to have these tendencies on an inner mesh only. For a coarse domain, it could be removing 16 3d arrays that are not required.

Reply:
@davegill We can do that.